earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -229,6 +237,7 @@ StorageManager<AssetType>::Get(
       // The file has changed on disk.
 
       // Drop the current entry from the cache.
+      cache.setPruneConditionVal(DetermineIfPrune());
```

Comment:
In this case we don't want to call `DetermineIfPrune()` at all. We're already passing `false` for the `prune` option - we can just leave that. What happens here is that we're removing something from the cache and then adding something else a little lower. There's no point in pruning twice, so we skip pruning for the remove and only prune during the add.